docs/api.rst

```diff
@@ -826,15 +826,27 @@ Members
               on_member_remove(member)
 
     Called when a :class:`Member` leaves or joins a :class:`Guild`.
+    If :func:`on_member_remove` is being used then consider using :func:`on_raw_member_remove` which will be called regardless of the cache.
 
     This requires :attr:`Intents.members` to be enabled.
 
     :param member: The member who joined or left.
     :type member: :class:`Member`
 
+.. function:: on_raw_member_remove(payload)
```

Comment:
I believe in #636 we changed the structure to list raw events after "complete" events, i.e. this would need to be moved between `on_member_update` and `on_raw_member_update`.

Reply:
The merge conflict I had earlier messed up the order of the events sorry